bugs/mahrud/atomic_test/test.cpp

```diff
@@ -0,0 +1,39 @@
+#include <vector>
```

Comment:
These files don't report bugs, so they don't belong in the `bugs` directory, which we want to get rid of.  They also don't seem to have anything to do with Macaulay2, so why are they here?

Reply:
This is a test, but some files in `bugs/mahrud` are useful scripts for debugging. There didn't seem to be a better place to share them. Should I move them to `M2/BUILD/mahrud` instead?

Reply:
That would be good, so we  can work on eliminating `bugs` entirely.  As you have remarked, those old bug reports are mostly useless.  There's just one of them that I want to save, because it consists of test procedures that should be moved to something like `M2/INSTALL-manual-testing`.

Reply:
Moved. I've been using the `atomic`, `thread_local`, and `pthread` tests to check those capabilities on different machines, and I think are useful for making sure a bug on a specific machine isn't caused by some architecture dependent behavior.

Reply:
Yes, it's worthwhile to save those somewhere ...